src/main/java/core/basesyntax/service/impl/ReaderFileImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReaderFile;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class ReaderFileImpl implements ReaderFile {
+    @Override
+    public List<String> readFromFile(String fileName) {
+        List<String> listFileRead;
+        try {
+            listFileRead = Files.readAllLines(Paths.get(fileName));
```

Comment:
```suggestion
            return Files.readAllLines(Paths.get(fileName));
```
можна одразу повертати результат